dags/openshift_nightlies/scripts/install/hypershift.sh

```diff
@@ -43,30 +51,45 @@ setup(){
     export HOSTED_KUBECONFIG_NAME=$(echo $KUBECONFIG_NAME | awk -F-kubeconfig '{print$1}')-$HOSTED_NAME-kubeconfig
     export HOSTED_KUBEADMIN_NAME=$(echo $KUBEADMIN_NAME | awk -F-kubeadmin '{print$1}')-$HOSTED_NAME-kubeadmin    
     export KUBECONFIG=/home/airflow/auth/config
-    export ROSA_CLI_VERSION=$(cat ${json_file} | jq -r .rosa_cli_version)
-    if [[ ${ROSA_CLI_VERSION} == "master" ]]; then
-        git clone https://github.com/openshift/rosa
-	pushd rosa
-	make
-	sudo mv rosa /usr/local/bin/
-	popd
+    if [[ $INSTALL_METHOD == "osd" ]]; then
+        export OCM_CLI_VERSION=$(cat ${json_file} | jq -r .ocm_cli_version)
+        if [[ ${OCM_CLI_VERSION} == "master" ]]; then
```

Comment:
So what happens if someone doesn't use the master branch? It looks like it doesn't get cloned down at all or am I missing something? Same thing for the below ROSA_CLI_VERSION as well

Reply:
This is just an optional override, the task [image](https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/images/airflow-managed-services/Dockerfile#L16-L17) got a stable version of CLI already.

Reply:
Yep, I couldn't find a way to do `if` conditions in a Dockerfile, so we have the stable version pre-installed and override the binary.

Reply:
I guess my question was more. What if I get a test branch from someone that I need to test. It would be convenient if maybe I can just pass in the full git url or something as an override. Thoughts?

Reply:
Have we ever used any custom branch or fork for CLIs before or how often we do such overrides ? just one less variable to worry :)

Reply:
So we have used custom branches for rosa during some early testing. My only thought here is that given hypershift is new and shiny we may end up in a position where we suddenly need a flag or something that isn't implemented yet. It also sets us up for the future. That said, I'm ok with no worrying about that in this PR and instead doing a followup to allow it for both rosa/ocm so we're covered in the future.

Reply:
Okay lets do that, its good to have that provisioned and also for `hypershift` CLI.
I will work on all 3 of them on a separate pull request and keep this PR simple for OSD implementation only.